RubberduckTests/Inspections/GeneralInspectionTests.cs

```diff
@@ -24,13 +24,27 @@ public void InitResources()
             RubberduckUI.Culture = Thread.CurrentThread.CurrentUICulture;
         }
 
+        private static List<Type> GetAllBaseTypes(Type type)
```

Comment:
this could be somewhat cleaner as an `IEnumerable<Type>` where you'd `yield` the separate results.